lib/category-name-converter.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
```

Comment:
@scottnath It's a best practice to contain the `'use strict';` statement within a function. Unintentional changes to the global scope are less of a problem in Node, but it certainly doesn't hurt to be as careful as you would be for browser js. This goes for all the `'use strict'`; statements in this this pr.